parsers/GT.py

```diff
@@ -0,0 +1,137 @@
+# The arrow library is used to handle datetimes
+import arrow
+# The request library is used to fetch content through HTTP
+import requests
+# The pandas library is used to manipulate real time data
+import pandas as pd
+
+tz_gt = 'America/Guatemala'
+
+MAP_GENERATION = {
+    'coal': 'Turbina de Vapor',
+    'gas': 'Turbina de Gas',
+    'hydro': u'Hidroeléctrica',
+    'oil': 'Motor Reciprocante',
+    'solar': 'Fotovoltaica',
+    'wind': u'Eólico',
+    'geothermal': u'Geotérmica',
+}
+
+
+def fetch_hourly_production(country_code, obj, hour, date):
+    
+    #output frame
+    data = {
+        'countryCode': country_code,
+        'production': {},
+        'storage': {},
+        'source': 'www.amm.org.gt',
+    }
+    
+    #Fill datetime variable
+    data['datetime'] = arrow.get(date, 'DD/MM/YYYY').replace(hour=hour).datetime   
```

Comment:
`date` here is a string with the datetime formatted in the local tz (if I understood correctly)
Thus, the arrow object you create will have no timezone associated because it's not specified anywhere (it can't come from the string). By default it will be interpreted as UTC, which is wrong because the formatted date is a date with the `tz` timezone.